RELEASE.md

```diff
@@ -52,4 +55,4 @@ The following steps should be always run, regardless of whether an automated or
    1. Mark this version of Stencil as 'released' in JIRA on the 'Releases' page.
```

Comment:
Something we forgot to do yesterday since we were all together:
```suggestion
   1. Call a `code-freeze` in the Stencil team channel
   1. Mark this version of Stencil as 'released' in JIRA on the 'Releases' page.
```

Reply:
Oh yeah - I just clicked the wrong 'Step 1.' for this comment 🤦